pkg/skaffold/survey/survey.go

```diff
@@ -62,6 +63,14 @@ func New(configFile string) *Runner {
 }
 
 func (s *Runner) DisplaySurveyPrompt(out io.Writer) error {
+	// TODO(nkubala): remove after v1.14.0 is released
+	currentVersion, err := version.ParseVersion(version.Get().Version)
+	if err == nil {
+		nextRelease, _ := version.ParseVersion("v1.14.0")
```

Comment:
This should check for error, or use semver.MustParse.

Reply:
updated